docs/f-droid.md

```diff
@@ -0,0 +1,46 @@
+# Adding F-Droid repositories
+
+To add an F-Droid repository you need at least the URL and a public key.
```

Comment:
Also point out that fdroid repos can be added manually in the app.  This is just to have images with fdroid repos already included.